### PR TITLE
Add CentOS8 image to cPouta

### PR DESCRIPTION
**Pull request by martbhell**
Also document in a table the default username.
Could not get "mkdocs build" to work so haven't tested how this looks.
Does not show up in the list of previews either.

### CODE REVIEW

**Comment by penikune** [comment]
I'll try to build this branch on Rahti.

File: docs/cloud/pouta/images.md [hunk 1]
```diff
@@ -34,9 +34,30 @@ keep additional packages at a minimum.
...
+
+### CentOS-8
+Currently in a *tech preview* because we have found some
+[minor issues](https://bugs.centos.org/view.php?id=16948) with it.
+
+CentOS is the community version of Red Hat Enterprise Linux (RHEL).
+This image is exactly as provided by CentOS and we have not modified it
+in any way.
+
+Note that CentOS-8 has `centos` as default username instance of `cloud-user`,
```
Comment (Aegil) [inline docs/cloud/pouta/images.md]: instance /instead

Comment (martbhell) [reply docs/cloud/pouta/images.md]: Thank you, adjusted that.

**Comment by junousi** [comment]
```
$ pip install grip
$ grip docs/cloud/pouta/images.md
```

...looked good to me. Want to take a look? Can approve after that.

There is a stray /* openstack image list --public */ that could be fixed or removed but this core change anyway is fine.

**Pull request merged**